templates/default/etc/systemd/system/knotx.service.erb

```diff
@@ -1,30 +1,30 @@
 [Unit]
-Description=<%= @knotx_id %>
+Description=<%= @id %>
 After=network.target
 
 [Service]
-EnvironmentFile=<%= @knotx_root_dir %>/knotx.conf
-ExecStart=/usr/bin/java \
-  -Dlogback.configurationFile=<%= @knotx_root_dir %>/logback.xml \
-  -Dvertx.cacheDirBase=<%= @knotx_root_dir %>/.vertx \
+EnvironmentFile=<%= @home_dir %>/knotx.conf
+ExecStart=<%= @java_home %>/bin/java \
+  ${KNOTX_GENERAL_OPTS} \
```

Comment:
I'm not sure if parenthesis is required here

Reply:
As far as I can see it's ok to use both `${VAR}` and `$VAR` in the unit file. Both will be understood w/o isssues: https://coreos.com/os/docs/latest/using-environment-variables-in-systemd-units.html

Reply:
Official systemd docs say it's ok: https://www.freedesktop.org/software/systemd/man/systemd.service.html

Reply:
After giving it a 2nd thought the `${VAR}` vs `$VAR` case matters a lot and systemd template was adjusted as you suggested.